docs/content-services/components/document-list.component.md

```diff
@@ -90,7 +90,6 @@ Displays the documents from a repository.
 | sorting | `string[] \| `[`DataSorting`](../../../lib/core/src/lib/datatable/data/data-sorting.model.ts) | ['name', 'asc'] | Defines default sorting. The format is an array of 2 strings `[key, direction]` i.e. `['name', 'desc']` or `['name', 'asc']`. Set this value only if you want to override the default sorting detected by the component based on columns. |
 | sortingMode | `"server" \| "client"` | "server" | Defines sorting mode. Can be either `client` (items in the list are sorted client-side) or `server` (the ordering supplied by the server is used without further client-side sorting). Note that the `server` option _does not_ request the server to sort the data before delivering it. |
 | stickyHeader | `boolean` | false | Toggles the sticky header mode. |
-| thumbnails | `boolean` | false | Show document thumbnails rather than icons |
```

Comment:
I suppose  `display` on line 69 should also be removed

Reply:
yes, it was display to remove , not thumbnails